compiler/test/dotty/tools/vulpix/RunnerOrchestration.scala

```diff
@@ -164,7 +164,9 @@ trait RunnerOrchestration {
      */
     private def createProcess: Process = {
       val url = classOf[ChildJVMMain].getProtectionDomain.getCodeSource.getLocation
-      val cp = Paths.get(url.toURI).toString + JFile.pathSeparator + Properties.scalaLibrary
+      val cp = Paths.get(url.toURI).toString +
+        JFile.pathSeparator + Properties.scalaLibrary +
+        JFile.pathSeparator + Properties.dottyLibrary
```

Comment:
I had to add the dottyLibrary to runtime of the test here, because otherwise the run_staging tests would fail on windows (as it would be fail trying to access `scala.quoted.runtime.Patterns.patternType` after the other changes). Not sure why this problem was windows only, but the way classpath is reconstructed from classloader in the staging compiler is a heuristic, so that might be the reason.